### ISSUE TITLE
Add a link to the keyboard help button in the shortcut paragraph

### ISSUE THREAD

**Issue by zepumph** [phetsims/ohms-law#99]
Promoted from TODO in OhmsLawSceneSummaryNode.js

@jessegreenberg how do we add a shortcut in a description? Have we done this before?

**Comment by terracoda** [comment]
@zepumph, I'd like to reconsider this idea. I originally thought that it would be helpful for users to have a normal anchor link that they could immediately follow if they wanted to. That would, however, add a "Keyboard Shortcuts" link to the Tab Order as the first item. I am not sure that is desirable.

I think the description on where to look for the Keyboard Shortcuts is actually enough. Users will find the Keyboard Shortcuts button on a full scan of the sim near the end. Until we know more, I think this works fine. And nobody has complained yet that they cannot find the keyboard help information.

**Comment by jessegreenberg** [comment]
> That would, however, add a "Keyboard Shortcuts" link to the Tab Order as the first item.

Good point, that would be confusing for sighted users. Lets close this issue for now. @terracoda if you decide that this link is beneficial let us know and we can work on adding general support for this.

**Comment by terracoda** [comment]
@jessegreenberg, will do. Thanks for closing.
